pom.xml

```diff
@@ -398,14 +399,36 @@
                 <version>${project.version}</version>
             </dependency>
             
+            <dependency>
+                <groupId>org.slf4j</groupId>
+                <artifactId>slf4j-api</artifactId>
+                <version>${slf4j.version}</version>
+            </dependency>
+
             <dependency>
                 <groupId>org.junit</groupId>
                 <artifactId>junit-bom</artifactId>
                 <version>5.9.1</version>
                 <type>pom</type>
                 <scope>import</scope>
             </dependency>
+            
+
+            <dependency>
+                <groupId>org.jboss.arquillian</groupId>
+                <artifactId>arquillian-bom</artifactId>
+                <version>1.7.0.Alpha10</version>
+                <type>pom</type>
+                <scope>import</scope>
+            </dependency>
 
+            <dependency>
```

Comment:
this dependency is already declared line 408.
Maven build generate a warning:
```
[WARNING] 'dependencyManagement.dependencies.dependency.(groupId:artifactId:type:classifier)' must be unique: org.junit:junit-bom:pom -> duplicate declaration of version 5.9.1 @ jakartatck:project:10.0.0-SNAPSHOT, /Volumes/workspace/dev/sources/open-sources/jetty/jakartaee-tck/pom.xml, line 425, column 25
```

Reply:
Thanks for noticing. Removed the same.